setup.py

```diff
@@ -141,5 +133,11 @@ def _unavailable(self, e):
     test_suite='zope.i18nmessageid.tests.test_suite',
     zip_safe=False,
     cmdclass={'build_ext': optional_build_ext},
-    **extra
+    ext_modules=ext_modules,
+    extras_require={
+        'testing': [
+            'nose', 'coverage'
```

Comment:
Do we actually allow `nose` as a test runner here?  AFAIU nose is deprecated upstream.